torch/csrc/jit/symbolic_script.cpp

```diff
@@ -178,11 +395,22 @@ const std::vector<std::string> functions = {
                    index: int):
             self_size = self.size()
             def backward(grad_output):
-                grad_self = torch.select_backward(grad_output, self_size, dim, index)
+                grad_self = ___select_backward(grad_output, self_size, dim, index)
                 return grad_self, None, None
 
             return torch.select(self, dim, index), backward
 
+        #def slice(self,
```

Comment:
Is this suppose to be commented?

Reply:
Yea `slice` AD triggers a test failure in TestJit of `test_distributions`. That's why I commented it out. I will make sure this gets fixed before merging. :)

Reply:
Based on our discussions offline, I will leave `aten::slice` commented here for now. Since the way to trigger this TODO in graph_executor is really rare, I think XLA can work around it by uncommenting in a patch. (There's a patch in current xla anyway).